README.md

```diff
@@ -113,11 +116,11 @@ The sample is configured by modifying the file [appsettings.placeholder.json](ap
 
 ### Configuring appsettings.json
 
-OSIsoft Cloud Services is secured by obtaining tokens from its identity endpoint. Client-credentials clients provide a client application identifier and an associated secret (or key) that are authenticated against the token endpoint. You must replace the placeholders in your `appsettings.json` file with the authentication-related values from your tenant and a client-credentials client created in your OCS tenant.
+AVEVA Data Hub is secured by obtaining tokens from its identity endpoint. Client-credentials clients provide a client application identifier and an associated secret (or key) that are authenticated against the token endpoint. You must replace the placeholders in your `appsettings.json` file with the authentication-related values from your tenant and a client-credentials client created in your ADH tenant.
 
 ```json
 {
-  "Resource": "https://dat-b.osisoft.com",                          # This is the base OCS URL being used
+  "Resource": "https://uswe.datahub.connect.aveva.com",                          # This is the base ADH URL being used
```

Comment:
looks like the formatting jumped with the extra characters in the url. can we get the comment realigned?

Reply:
nice catch!